docs/troubleshooting.rst

```diff
@@ -23,17 +31,16 @@ Then it is likely that you are using the older or newer version of the (simulate
 
 Mismatch in Inputs 
 ---------------------
-There are several files which are passed into Sorcha. These are the 
-orbit file, the physical parameter file and an optional cometary parameter file. Each provide
-specific information about the synthetic population that is being analysed.
-
-Within these files, it is necessary to specify an entry for every object. The Sorcha 
-code will run a check to ensure that all entries have an associated 
-orbit and physical/cometary parameter value, so if you get an error like::
+There are several files associated with the synthetic small bodies  which are passed into Sorcha. These are
+the orbit file, the physical parameter file and an optional complexy parameters file and optional ephemeris 
```

Comment:
"physical **parameters** file" and also "and an optional **complex** parameters file"